librz/core/gdiff.c

```diff
@@ -447,6 +447,13 @@ static int graph_construct_nodes(RzCore *core, RzCore *core2, RzAnalysisFunction
 				core2->config = core->config;
 
 				char *modified = rz_core_cmd_strf(core2, "pdb @ 0x%08" PFMT64x, bbi->diff->addr);
```

Comment:
1. Check if the `modified` is NULL before passing to the `strcmp()`
2. Is it possible to use the API instead of calling the `pdb` command while you are on it?

Reply:
i just checked the pdb api, and i'm definetely not going to use the native one for this. there is no way to get a text unless i use rz_cons which is a mess.